pkg/kubelet/apis/kubeletconfig/v1beta1/types.go

```diff
@@ -284,172 +391,268 @@ type KubeletConfiguration struct {
 	//   "none":               do nothing.
 	// Generally, one must set --hairpin-mode=hairpin-veth to achieve hairpin NAT,
 	// because promiscuous-bridge assumes the existence of a container bridge named cbr0.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// it may require a node reboot, depending on the network plugin.
 	// Default: "promiscuous-bridge"
 	// +optional
 	HairpinMode string `json:"hairpinMode,omitempty"`
 	// maxPods is the number of pods that can run on this Kubelet.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// changes may cause Pods to fail admission on Kubelet restart, and may change
+	// the value reported in Node.Status.Capacity[v1.ResourcePods], thus affecting
+	// future scheduling decisions.
```

Comment:
nit: Also warn that increasing the value may have a negative performance impact.

Reply:
Done, thanks.